lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/LionWebModelConverter.kt

```diff
@@ -702,11 +746,81 @@ class LionWebModelConverter(
         return kNode
     }
 
+    private fun <T : Any> maybeInstantiateSpecialObject(kClass: KClass<T>, data: Node): T? {
+        return when (kClass) {
+            Issue::class -> {
+                Issue(
+                    attributeValue(data, data.classifier.getPropertyByName("type")!!) as IssueType,
```

Comment:
I would cosider using Issue::type.name and similar values, to protect against typos